db/migrate/20220518120740_create_statement_line_items.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateStatementLineItems < ActiveRecord::Migration[6.1]
+  def change
+    create_table :statement_line_items do |t|
+      t.references :statement
+      t.references :participant_declaration
+
+      t.text :state
```

Comment:
```suggestion
      t.text :state, null: false # possibly default: "eligible"?
```

Reply:
i can stick the null protection in but don't think setting a default would be wise in case someone forgets to set it